BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -513,6 +509,14 @@ extension WalletViewController {
     }
   }
   
+  func refreshBalance() {
```

Comment:
Naming is confusing as there is no action in this method that is refreshing the balance. `updateWalletHeader` or something along the lines of that would be clearer.